### ISSUE TITLE
Show history of project quota edit requests

### ISSUE THREAD

**Issue by jleach** [bcgov/platform-services-registry#856]
**Describe the issue**

When evaluating project quota edit requests its is handy for each request to be evaluated against previous requests. For example, when asking for 8->16 CPU cores did the project first ask for smaller increments. This is important because it shows they may be trying to work in as small a footprint as possible before growing. It is also important to know who in the team approved or rejected a request so that follow up can be done and it adds context.

**Additional context**

Blocked by an additional ticket to create mocks (high fed figma)

**Definition of done**

As an administrator of the registry, I would like to be able to click on a project and see an ordered list of history quota requests showing what the request was for, the date it was submitted, the date it was processed, and weather it was approved or rejected.

**Out of scope. Future Vision**

**Comment by faisal-hamood** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Experience-5bb7c5ab4b5806bc2beb9d15/issues/bcgov/platform-services-registry/856?planning-poker) with Zenhub @okanji

**Comment by lukegonis** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Experience-5bb7c5ab4b5806bc2beb9d15/issues/bcgov/platform-services-registry/856?planning-poker) with Zenhub @alexandralloyd @funtigr @Kolezhanchik @okanji

**Comment by alexandralloyd** [comment]
https://www.figma.com/file/llP34TwOrBUjUGey9PUu6c/Platform-Registry?node-id=300-25284&t=NvAj5ld6o54ZFwtG-0

Please see Dashboard Home > Private Cloud > Products > Request History

**Comment by alexandralloyd** [comment]
We will need to make a decision on if we should have a scroll or a pagination for longer request histories
